src/components/ArticlesList.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState, useEffect } from "react";
+import { getArticles } from "../api";
+import ArticleCard from "./ArticleCard";
+import "./ArticlesList.css";
+
+const ArticlesList = () => {
+  const [articles, setArticles] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [error, setError] = useState(null);
+
+  useEffect(() => {
+    getArticles()
+      .then((data) => {
+        //console.log( data); // Object does not return body
+        setArticles(data);
+        setIsLoading(false);
+      })
+      .catch((error) => {
+        setError("Failed to fetch articles.");
+        setIsLoading(false);
+      });
+  }, []);
+
+  if (isLoading) return <div className="loading">Loading...</div>;
+  if (error) return <div className="error">{error}</div>;
+
+  return (
+    <div className="articles-list">
+      {articles.map((article) => (
```

Comment:
as it's a list of articles, it might be better to map through the articles with in a <ul> and return a <li> in the map